src/insights/funding.ts

```diff
@@ -32,3 +32,18 @@ export interface QueryGetFundingHistoryRequest {
   limit?: number;
   offset?: number;
 }
+
+export interface QueryGetFundingHistoryGraphDataRequest {
+  market: string;
+  period: number;
```

Comment:
Maybe can add a comment stating that this is the duration (in number of days) that you want the query to return results for.